capstone/src/main/java/com/google/sps/servlets/UpdateAssociationServlet.java

```diff
@@ -48,6 +54,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     return comments;
   }
 
+  /**
+   * Removes all previous association results from datastore
+   *
+   * @param datastore the datastore client object
```

Comment:
Since this entire class is centered around reading from data store, it may be easier to make the data store service an instance variable. This will also make it easier since you won't have to pass the data store objects as an argument into each function. When dealing with multiple objects, it makes sense to limit the scope of each object and pass them to helper functions, but in this case it seems that every function uses data store

Reply:
Moved this to an init function